Test/git-issues/github-issue-2500.dfy

```diff
@@ -0,0 +1,31 @@
+// RUN: %dafny_0 /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module A {
+  trait {:termination false} Trait {
+    function method TotallyNotZero() : (ret: int)
+      ensures ret != 0
+  }
+}
+
+module B {
+
+  import opened A
+
+  class Class extends Trait {
+    constructor() {}
+    function method TotallyNotZero() : (ret: int)
+      // Missing: ensures ret != 0
+    { 
+      0 
+    }
```

Comment:
Could you give a version of this example that doesn't use `{:termination false}`?